.github/workflows/test-internal-prs.yml

```diff
@@ -130,14 +130,14 @@ jobs:
         run: yarn license
 
   unit:
-    uses: aws-amplify/amplify-ui/.github/workflows/reusable-unit.yml@main
+    uses: ./.github/workflows/reusable-unit.yml
```

Comment:
or should we do `aws-amplify/amplify-ui/.github/workflows/reusable-unit.yml@categorical-packages/main` to be more strict?

Reply:
I think either works, but let's just use absolute path `@categorical-packages/main` to be extra safe.